src/content/blog/ga_qas.md

```diff
@@ -0,0 +1,233 @@
+---
+title: An Introduction to GA-QAS
+author: Vu Tuan Hai
+day: 21
+month: 5
+year: 2024
+tags:
+  - quantum architecture search
+  - python
+  - qiskit
+  - genetic-algorithm
+---
+
+In various quantum optimization problems, choosing the right ansatz is a critical point that will affect the accuracy of the result. Many template ansatzes have been proposed, such as Graph Ansatz and EffecientSU2, … but they are limited in their applicability. Hence, we've created a search engine called **GA-QAS** (Genetic Algorithm for Quantum Architecture Search) to aid in the discovery of the right ansatz. In this post, we will guide you on how to use it efficiently.
+
+Required packages:
+
+- qiskit 1.0+
+- matplotlib
+- tdqm
+
+## Now, let’s start!
+
+# Step 1. Define problem (W state preparation)
+
+<figure>
+    <img src='/public/images/ga-qas/ga-qas1.png' width="600" style="margin: auto" alt='description' />
```

Comment:
Since the caption is now associated with this image I think we can provide a null alt attribute to denote to screen readers that the image can be ignored. More info [here](https://www.boia.org/blog/should-you-include-alt-text-for-pictures-with-captions).

```suggestion
    <img src='/public/images/ga-qas/ga-qas1.png' width="600" style="margin: auto" alt='' />
```